dev/com.ibm.ws.install/src/com/ibm/ws/install/internal/ResolveDirector.java

```diff
@@ -299,7 +305,13 @@ RepositoryConnectionList getRepositoryConnectionList(Collection<String> featureN
                     }
                     throw ExceptionUtils.createByKey(e, "ERROR_FAILED_TO_CONNECT");
                 }
-                throw ExceptionUtils.create(e, featureNames, false, proxy, true);
+                String edition = this.product.getProductEdition();
```

Comment:
As in other comments, use getProductId() to check for io.openliberty.  Don't need the "Open Liberty" or " edition." strings.